### ISSUE TITLE
Abnormality of the cerebral venous sinuses & Central venous sinus thrombosis

### ISSUE THREAD

**Issue by DLewisSmith** [obophenotype/human-phenotype-ontology#6743]
**Preferred term label:**
Abnormality of the cerebral venous sinuses
Central venous sinus thrombosis

**Synonyms**

**Definition (free text, please give PubMed ID)**
Abnormality of the cerebral venous sinuses:
An abnormality of the venous sinuses that drain cerebral veins to the internal jugular veins.

Central venous sinus thrombosis:
An intracranial thrombosis of the venous sinuses. These typically present with headache, seizures or venous stroke secondary to raised cerebral venous pressure. Cerebral venous sinus thromboses usually affect larger areas of brain parenchyma than those affected by cerebral vein thromboses. (29923367)

I have not suggested terms for alternative types of Abnormality of the cerebral venous sinuses because these are concepts such as sinus hypoplasia and stenosis which are less well defined and exist more as differential diagnoses of CVST.

**Parent term (use hpo.jax.org/app)**
Abnormality of the cerebral vasculature HP:0100659

Central venous sinus thrombosis -> Abnormality of the cerebral venous sinuses
Central venous sinus thrombosis -> Venous thrombosis HP:0004936

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0002-1735-8178

**Comment by pnrobinson** [comment]
@DLewisSmith  -- these are not typical veins, but should the parent actually be 
https://hpo.jax.org/app/browse/term/HP:0012480
Abnormal cerebral vein morphology HP:0012480

**Comment by DLewisSmith** [comment]
Agreed, that seems better. Thanks @pnrobinson

**Comment by DLewisSmith** [comment]
I would also modify the definition of Abnormality of the cerebral venous sinuses to:
An abnormality of the venous sinuses that drain blood from the cerebral veins and CSF from the arachnoid granulations to the internal jugular veins.

**Comment by pnrobinson** [comment]
adding as Abnormal cerebral venous sinus morphology

**Comment by pnrobinson** [comment]
I am adding the Abnormal dural venous sinus morphology is a synonym

**Comment by pnrobinson** [comment]
added terms
